scripts/run_mission1_robustness.py

```
@@ -0,0 +1,616 @@
+    return {
+        "robot_id": config.robot_id,
+        "camera_id": camera_id,
+        "target_class": config.target_class,
+        "metrics_path": ROOT / config.output_dir / f"{config.name}_metrics.json",
+    }
+
+
+def start_process(cmd: list[str], log_path: Path, env: dict[str, str], cwd: Path) -> subprocess.Popen:
+    log_path.parent.mkdir(parents=True, exist_ok=True)
+    log_handle = open(log_path, "w", encoding="utf-8")
+    proc = subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)